### ISSUE TITLE
Cannot find module './App.vue'

### ISSUE THREAD

**Issue by mludi** [vuejs/vue-cli#1215]
### Version
3.0.0-beta.6

### Reproduction link
[https://github.com/mludi/vue-cli-issue](https://github.com/mludi/vue-cli-issue)

### Steps to reproduce
3.0.0-beta.9

Clone the project, cd into run and run `npm install`

### What is expected?
Visual Studio Code shows a warning in main.ts and router.ts, it cannot find module './xx.vue'

### What is actually happening?
There is already a shims.d.ts file, but there is something wrong with the content.

---
With some modification on the shims.d.ts file it's working fine, see:

```
declare module '*.vue' {
    import Vue from "vue";
    export default Vue;
}
```

**Comment by forsartis** [comment]
Same as #1198 and already fixed in commit 51c8090.

**Comment by mludi** [comment]
Oh sorry haven‘t seen that. Thank you!
